app/src/main/java/com/cpr3663/cpr_scouting_app/Settings.java

```diff
@@ -221,7 +221,7 @@ public void onClick(View view) {
                 Globals.CurrentPrefTeamPos = spinner_PrefTeamPos.getSelectedItemPosition();
                 Globals.spe.putInt(Constants.SP_PREF_TEAM_POS, Globals.CurrentPrefTeamPos);
                 Globals.spe.apply();
-                Exit();
+                finish();
```

Comment:
Cool makes sense why and how you are doing this. But is there any reason to not just delete the function "Exit"? if not do so.

Reply:
Exit is deleted, hence it's in red.

Is your asking why change it, themes be sure finish is more "expected" in android and runs any cleanup, etc.

Or did you mean something else?

Reply:
Not that the function below.

Reply:
done.